opacus/tests/dp_layers_test.py

```diff
@@ -8,6 +8,7 @@
 from torch.nn import LSTM
 from torch.nn.modules.activation import MultiheadAttention
 from torch.testing import assert_allclose
+from torch.nn.utils.rnn import pad_sequence, pack_padded_sequence
```

Comment:
After landing #120  we have rewritten these tests to use Hypothesis. Can we rebase this on top of Master and update these tests?

I would do it by making a new bool flag `packed_input`, and calling `_gen_packed_data` on the input and `pad_packed_sequence` on the output if it's True (I think we are going to need to latter to use `torch.testing.all_close`? I doubt it's smart enough to accept PackedSequences directly). Everything else should be the same I think? 

Note: you can have another flag for "sorted_sequence", and then in the body of the function just check if `pack_sequence = True`. If not, disregard that flag

Reply:
I have updated the tests @Darktex . So, I have instead added a flag "packed_input_flag" which is 0 for standard input, 1 for sorted PackedSequence, and 2 for unsorted PackedSequence in order to keep the configuration space small.
I had to adapt the ```DPModules_test``` class within ```common.py``` for testing PackedSequence data. See the ```DPModules_test._check_packed_sequence``` method.

Reply:
Works great!